pkg/agent/proxy/metrics/metrics.go

```diff
@@ -0,0 +1,117 @@
+package metrics
+
+import (
+	"sync"
+
+	kmetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog"
+)
+
+var (
+	once sync.Once
+
+	subsystemProxy = "antrea_proxy"
+
+	SyncProxyLatency = kmetrics.NewHistogram(
+		&kmetrics.HistogramOpts{
+			Subsystem:   subsystemProxy,
+			ConstLabels: map[string]string{"ipFamily": "v4"},
+			Name:        "sync_proxy_rules_duration_seconds",
+			Help:        "SyncProxyRules latency in seconds of Antrea Proxy",
+			Buckets:     kmetrics.ExponentialBuckets(0.001, 2, 15),
```

Comment:
curious about whether there is a specific reason for using these buckets instead of the default ones (https://github.com/kubernetes/component-base/blob/e05af43f81e23f34bfe3a6d38be47bd36ca6841c/metrics/histogram.go#L25) ?

Reply:
These parameters actually follow the kube-proxy. After reviewing the code you pointed, I thought the default bucket is reasonable enough actually.
Removed the initialization of this field.